### PR TITLE
#167193186 Deploy Base Frontend to Heroku

### PR DESCRIPTION
**Pull request by BrianSerem**
## Description ##
To test our application in an environment resembling a production server, we need to deploy our application to Heroku. This pull request ensures our application in deployed. [Here](https://landville-frontend.herokuapp.com/) is the application in action.

## Type of change ##
Please select the relevant option
- [ ] Bug fix(a non-breaking change which fixes an issue)
- [x] New chore(a non-breaking change which adds functionality)
- [ ] Breaking change(fix of a feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

## How Has This Been Tested? ##
Please describe the tests that you ran to verify your changes
- [ ] End to end test
- [ ] Integration test
- [ ] unit test

## Checklist: ##
- [x] My code follows the style guidelines of this project
- [x] I have linted my code prior to submission
- [x] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing tests pass locally with my changes

## Related Pivotal Tracker stories ##
[#167193186](https://www.pivotaltracker.com/story/show/167193186)

### CODE REVIEW

**Review by kelvinndmo — APPROVED** [review]

**Review by akram256 — APPROVED** [review]
LGTM

**Comment by akram256** [comment]
Hey @BrianSerem, Please resolve the merge conflicts

**Review by MandelaK — APPROVED** [review]

**Review by joelethan — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -3,35 +3,41 @@
   "version": "0.0.0",
   "scripts": {
     "ng": "ng",
-    "start": "ng serve",
+    "start": "node server.js",
```
Comment (joelethan) [inline package.json:6]: Hello @BrianSerem just for clarity, why did we change this from `ng serve` to `node server.js`

Comment (BrianSerem) [reply package.json:6]: This is the start command needed by Heroku. Don't worry, our good old `ng serve` will still work.

**Review by patrickf949 — APPROVED** [review]

**Review by JamesMudidi — APPROVED** [review]
Hello @BrianSerem, I have tested the application deployed on Heroku and it works fine

**Comment by malep2007** [comment]
@BrianSerem, please rebase your PR for merging.

**Comment by malep2007** [comment]
@BrianSerem, please rebase and fix conflicts.

**Pull request merged**